src/main/java/core/basesyntax/service/FileReaderService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface FileReaderService {
+    List<String> readFile(String filePath);
```

Comment:
```suggestion
    List<String> readFile(String fileName);
```

Reply:
not fixed

Reply:
Please fix this parameter name as Elena advised